### ISSUE TITLE
The Document Outline

### ISSUE THREAD

**Issue by SqueeG** [SqueeG/awesomeTome#16]
So @Lokathor and I talked a little bit and using his suggestions I made some alterations to the folder and file structure of the project.

Lokathor made a PDF with his idea of the Tome's chapter structure. It's the [tome-ref-doc.pdf](https://github.com/SqueeG/awesomeTome/tree/master/current-source) located in the current-source folder.

Let's get a chapter layout settled on in the next day or two.

**Comment by Tarkisflux** [comment]
Spells may as well be after magic, instead of after unrelated advice on running the game and appendices. Appendices should either be at the actual end or right before the list of tables. The license can be the last or first thing, but before the table index is weird. Prestige classes should either be with the NPC classes before the monsters (if we're writing them in as optional based on world fluff) or right after the PC classes (if they're always an option regardless of world fluff). I'd put NPC classes before monsters as well, but that's not as big as the other bits I think.

That all said... why 1 giant book instead of 2 or 3?

**Comment by SqueeG** [comment]
@Lokathor just put that out as his suggested course. Preferring an all-in-one so we can do in-document referencing for spells and the like.

That outline isn't concrete though. And (for now) I would prefer working on the material separately, with the PHB being of highest priority. But I do see merit in an AoI as well.

**Comment by Lokathor** [comment]
Spells are going to take up hundreds and hundreds of pages, so I put them as an appendix. You don't generally read through all the spells in order like you do when browsing feats or equipment.

The OGL should probably go farther toward the end, sure.

Prestige Classes could be a Main chapter or an Appendix chapter, either works. NPC Classes are separated and after Monsters because even the GM is not likely to use them. The NPC Classes are honestly that useless.

**Comment by Tarkisflux** [comment]
So... why are we even including the NPC classes?

**Comment by Lokathor** [comment]
Mostly because its like 5 pages. A GM might use it, but most NPCs in towns go un-statted or just have PC class levels, in my experience.

**Comment by ExplosiveRunes** [comment]
For a minimal time investment we could probably introduce one or two more Tome style 5 level NPC classes to go with the Tome Warrior and Tome Adept to give a little more variety to mooks. If we also had a thief-lite (bandit?) and a cleric-lite (acolyte?) we could have NPC versions of the classic Fighting-Man/Magic User/Thief/Cleric line-up.

**Comment by Tarkisflux** [comment]
Bandit and Acolyte would fill the robber / healbot lineup nicely, and if they're pegged to CR/character level instead of class level we might be able to turn the NPC class chapter into minor monster bolt on / template chapter stuff and make it less a waste of time. So sure, let's try to do that. We can always release without though, just in case we forget about it / don't care anymore when it comes due.

**Comment by Tarkisflux** [comment]
We have a outline for a single book (as discussed here) and an outline for a triplet of books (as presented in the folder structure), neither of which seems contentious any longer. Issue closed.

**Comment by Lokathor** [comment]
No, you can't end the issue there. Because we need to pick which format we're going with still. A single book and a book triplet probably won't have the same structure really, because even the content would probably be organized differently. So we have to actually pick which we're doing. Just putting all content all around and then having four master documents (Single Book, PHB, MCB, MM) that each whatever they think they should have is just asking for trouble down the line.

**Comment by SqueeG** [comment]
Ehhh.... I look at shit on my tablet and smart phone rather often, which means GIANT documents are a bitch to look at because they tend to bog down the PDF viewers.

On the other hand, spell referencing/linking and the like is complete win... so one master document would be great for that.

We should definitely remember to do a Quick-Reference Player's Guide at the end, though.

**Comment by Tarkisflux** [comment]
I don't see why we need to actually do two different formats for that, so much as two different sets of build files. If the chapters are otherwise the same between versions, and I would expect them to be aside from their order, it's pretty easy to maintain two different builds. You just make the individual chapters as build files, and then include only the chapters in your overall build file and call it a day. So the mega version and the split version are basically the same, just with the chapters all in one place or spread out across 3 different documents. 

Or am I missing something that would make that hard for some reason?
